docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1368,12 +1395,17 @@ Following is the beautified response of sample polling location request after th
 
 For the Import Operation, the polled status includes an indication of `$import` and the location of the OperationOutcome NDJSON files and the corresponding failure and success counts.
 
+For S3 exports, the bulk data feature may use presigned urls.  To enable presigned urls, the authentication type must be `hmac` and `fhirServer/bulkdata/storageProviders/(source)/presigned` must be `true`.  The urls become:
+
+```
+https://s3.appdomain.cloud/fhir-example/Patient_1.ndjson?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=fcEXbf9cc1ac49e99eEX77%2F20210228%2Fus%2Fs3%2Faws4_request&X-Amz-Date=20210EXT160538Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-Amz-Signature=5ecfc207546b9737fd38d4f0EXEX1c58f4f82976338a44c
+```
+
 Note, the deletion of an a job is split into two phases, ACCEPTED (202) response and DELETED (204).  202 is returned until the operation is stopped or removed, and then 204.
 
 By default, the exported `ndjson` file is configured with public access automatically and with 2 hours expiration time, the randomly generated secret in the path is used to protect the file. please note that IBM COS does not support expiration time for each single COS object, so please configure retention policy (e.g, 1 day) for the bucket if IBM COS is used. For both Amazon S3 and IBM COS, please remember that public access should never be configured to the bucket itself.
 
-Note: `fhirServer/bulkdata/isExportPublic` can be set to "false" to disable public access.
-      minio doesn't support object level ACL, so access token is always needed to download the exported `ndjson` files.
+Note: `fhirServer/bulkdata/storageProviders/(source)/exportPublic` can be set to "false" to disable public access. Also, *minio* doesn't support object level ACL, so access token is always needed to download the exported `ndjson` files.
```

Comment:
what happens if exportPublic is false and presigned is true?  can we still download the files?

Reply:
presigned still works, and we can download files.  Minio doesn't use acls like this, and it still works with presigned.

Reply:
per the previoius paragraph, I don't think IBM COS supports it either
> Please note that IBM COS does not support expiration time for each single COS object, so please configure retention policy (e.g, 1 day) for the bucket if IBM COS is used. For both Amazon S3 and IBM COS, please remember that public access should never be configured to the bucket itself.

makes me wonder if we should just retired that and recommend HMAC + presigned URLs for everyone